tests/ui/collapsible_span_lint_calls.rs

```diff
@@ -0,0 +1,26 @@
+[warn(clippy::collapsible_span_lint_calls)]
+
+fn lint_something() {
+    let msg = "warn about something";
+    let help = "this would help";
+    let note= "specificly note on this thing";
```

Comment:
Is this OK? Do I need to pay special attention to make a more semantic coherent message?